src/v/raft/state_machine_manager.h

```diff
@@ -84,12 +106,19 @@ class state_machine_manager final {
         }
     }
 
+    ss::future<> remove_local_state();
+
 private:
     using stm_ptr = ss::shared_ptr<state_machine_base>;
+    struct named_stm {
+        named_stm(ss::sstring, stm_ptr);
+        ss::sstring name;
+        stm_ptr stm;
+    };
 
     state_machine_manager(
       consensus* raft,
-      std::vector<stm_ptr> stms_to_manage,
+      std::vector<named_stm> stms_to_manage,
```

Comment:
should we make this an `absl::flat_hash_map` now? Or keep the struct and make it a `absl::flat_hash_set`?

Reply:
oh ignore me